Sources/ENCore/App/Features/ExposureController/ExposureStateStream.swift

```diff
@@ -73,10 +72,10 @@ final class ExposureStateStream: MutableExposureStateStreaming {
     func update(state: ExposureState) {
         currentExposureState = state
 
-        subject.send(state)
+        subject.onNext(state)
     }
 
     // MARK: - Private
 
-    private let subject = CurrentValueSubject<ExposureState?, Never>(nil)
+    private var subject = BehaviorSubject<ExposureState?>(value: nil)
```

Comment:
In case this never errors out, this should be a `BehaviorRelay` instead and then the `exposureState` should be a `Driver` instead of an `Observable`. Or even better yet, since you're starting with `nil` but don't expose an optional, it should really be a `PublishRelay` and then `exposureState` should be of type `Signal`.

Reply:
Thanks, made an internal ticket to clean this up